l1-contracts/src/core/Leonidas.sol

```diff
@@ -127,49 +112,31 @@ contract Leonidas is Ownable, TimeFns, ILeonidas {
    *          - Set the seed for the epoch
    *          - Update the last seed
    *
-   * @dev     Since this is a reference optimising for simplicity, we store the actual validator set in the epoch structure.
+   * @dev     Since this is a reference optimising for simplicity, we leonidasStore the actual validator set in the epoch structure.
```

Comment:
Whoops, a search a replace gone wrong. I am inclined to not fix this unless there are other more important stuff, since rerunning the ci tests for a comment will make me yearn for the urn.